webapp/components/LoginForm/LoginForm.spec.js

```diff
@@ -57,5 +58,20 @@ describe('LoginForm', () => {
         })
       })
     })
+
+    describe('Warning message is visible when caps lock is on', () => {
+      const wrapper = Wrapper()
+
+      it('Caps-lock warning message is hidden on page load', () => {
+        expect(wrapper.find('text.caps-warning').attributes('style')).toEqual('display: none;')
+      })
+
+      it('shows warning when caps-lock is press', async () => {
+        const formWrapper = wrapper.find({ ref: 'form' })
+        formWrapper.trigger('keydown', {key: 'CapsLock'})
```

Comment:
The one issue I am having with this test case is that I am not able trigger or emit an event to simulate the changing of the data value from false to true.